maps/mjolnir/bos/structures/statue.dm

```diff
@@ -1,41 +1,41 @@
 /obj/structure/memstatue
-	name = "statue stand"
-	desc = "You will get here one day. Or it's just a statue honouring some piece of tile."
-	icon = 'maps/torch/bos/icons/obj/structures/statue.dmi'
-	icon_state = "placeholder"
-	anchored = TRUE
+    name = "statue stand"
+    desc = "You will get here one day. Or it's just a statue honouring some piece of tile."
+    icon = 'maps/torch/bos/icons/obj/structures/statue.dmi'
+    icon_state = "placeholder"
+    anchored = TRUE
 
 /obj/structure/memstatue/roger
-	name = "Roger Blackmore statue"
-	desc = "A native of Tersten, a war hero who was head of the Tersten cell of the League of Struggle. Led the takeover of Tersten City, marched in the forefront of the rebels, blew himself up next to one of the army's military installations, posthumously a hero of the Sovereign Lordania."
-	icon_state = "shahid"
+    name = "Roger Blackmore statue"
+    desc = "A native of Tersten, a war hero who was head of the Tersten cell of the League of Struggle. Led the takeover of Tersten City, marched in the forefront of the rebels, blew himself up next to one of the army's military installations, posthumously a hero of the Sovereign Lordania."
```

Comment:
`who was head` - > `who was a head`(since we use the noun here) or `who was a leader`